src/helpers/pagination.ts

```diff
@@ -0,0 +1,30 @@
+interface PaginateProps {
+  items: unknown[];
+  pageNumber: number;
+  pageSize: number;
+}
+
+const DEFAULT_PAGE_SIZE = 10;
```

Comment:
Should we move this as a constant file separately?
So we can use that file as a global configuration.
I'm OK to keep in here also.
What do you think?

Reply:
Ok to me. But I'm not sure where should we put them. I put them into `config` in https://github.com/nimblehq/til-web/pull/18/commits/48d02e70933d005b05419a804c13c51635f8f8fd
Is it ok?